pyflakes/test/test_other.py

```diff
@@ -2073,6 +2073,45 @@ class A:
         class B: pass
         ''', m.UndefinedName)
 
+    def test_typeCommentsMarkImportsAsUsed(self):
```

Comment:
hah oh boy this class got abused for all sorts of things -- I just stuck this next to the other type annotation tests.

How about I split this up in a followup? there's a lot of diff churn already here and I think it would make sense for just a "mv" commit that's easy to review

Reply:
Created followup issue here: https://github.com/PyCQA/pyflakes/issues/402